config/routes.rb

```diff
@@ -1,6 +1,19 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  devise_for :users
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root "public_recipes#index"
```

Comment:
- [ ] Right now I am having some trouble trying to log in to the app. After signing up for the app, I try to log in and it does not allow me to go into it. If I try to access the routes on the search bar, it does not let me as the app thinks I am not authorized, so it would be nice if you could check on this!
- [ ] Because of the previously mentioned issue, I am not able to test the rest of the app, so it would be nice if you check it just before asking for a new review!

Reply:
@Lordkaito Thank you for the review. I also sent you a message on slack, hope it finds you well 🙏🏾